src/v2/view-builder/views/signin/SignInWithDeviceOption.js

```diff
@@ -73,5 +72,15 @@ export default View.extend({
     return {
       signInWithDeviceIsRequired: !!this.options.isRequired,
     };
+  },
+
+  postRender() {
+    if(this.options.isRequired) {
```

Comment:
To make the text dynamic, had to move over to the postRender method to use loc() localization method. Let me know if there is a easier way.